wagtail/project_template/project_name/settings/base.py

```diff
@@ -26,6 +26,7 @@
 INSTALLED_APPS = [
     'home',
     'search',
+    'media',
```

Comment:
Since the default media (file uploads) directory and URL is also named `media` (an old Django convention and also a Wagtail convention) - I am worried having an app named `media` will create a lot of collisions and/or confusion with this convention. For example, it is common to exclude `media/` from version control. Or newcomers might be wondering why there is python code in their `media` directory?

An alternate approach could be to change Wagtail's default MEDIA_ROOT and MEDIA_URL to `uploads` or something similar. However I would be worried about straying from the Django precedents of default `media` and `static` directories.

Reply:
Same as Vince, there are already two things called `media` in the Wagtail ecosystem so we need to think of another app name. RFC 44 proposed `assets`. I think we should get that RFC approved with a specific app name before we proceed.